js/main.js

```diff
@@ -0,0 +1,28 @@
+var $photoUrl = document.querySelector('.photo-url');
+var $imgSrc = document.querySelector('.img-source');
+
+$photoUrl.addEventListener('input', function (event) {
+  if ($photoUrl.value === '') {
+    $imgSrc.setAttribute('src', './images/placeholder-image-square.jpg');
+  } else {
+    $imgSrc.setAttribute('src', $photoUrl.value);
+  }
+});
+
+var $entryForm = document.querySelector('.data-form');
+$entryForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var $titleEntryValue = document.querySelector('#title').value;
+  var $photoUrlValue = document.querySelector('.photo-url').value;
+  var $notesEntryValue = document.querySelector('#notes').value;
```

Comment:
while this does work, I would recommend using the setup from the `js-forms` exercise where you use the `elements` property so that you don't have to query the dom for the inputs each time you submit the form.